core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1172,9 +1173,8 @@ public ServerBuilder clientAddressFilter(Predicate<InetAddress> clientAddressFil
 
     /**
      * Sets the default access logger mapper for all {@link VirtualHost}s.
-     * Unless you specify the access {@link Logger} by using {@link VirtualHostBuilder#accessLogger(Logger)}
-     * or {@link VirtualHost#accessLogger(Logger)},
-     * all of {@link VirtualHost} will have an access logger set by {@code mapper}
+     * the {@link VirtualHost}s which are not set by a user via {@link VirtualHostBuilder#accessLogger(Logger)}
```

Comment:
- the -> The
- How about `The {@link VirtualHost}s which do not have an access logger specified by a {@link VirtualHostBuilder} will ...`? I think it's unnecessary to mention setter methods here.